locale/de/foreman_scc_manager.po

```diff
@@ -123,7 +125,7 @@ msgstr ""
 msgid "Incremental Update"
 msgstr ""
 
-msgid "Incremental Update of 0 Content View Version(s) "
+msgid "Incremental Update of  Content View Version(s) "
```

Comment:
there should be a placeholder for the number of CVVs

Reply:
String was manually removed since it is from Katello.